src/Elastic.Transport/Components/Serialization/LowLevelRequestResponseSerializer.cs

```diff
@@ -2,33 +2,28 @@
 // Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information
 
-using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
-using System.IO;
 using System.Linq;
 using System.Text.Json;
 using System.Text.Json.Serialization;
-using System.Threading;
-using System.Threading.Tasks;
+
 using Elastic.Transport.Extensions;
-using static Elastic.Transport.SerializationFormatting;
 
 namespace Elastic.Transport;
 
 /// <summary>
-/// Default implementation for <see cref="Serializer"/>. This uses <see cref="JsonSerializer"/> from <code>System.Text.Json</code>.
+/// Default low level request/response-serializer implementation for <see cref="Serializer"/> which serializes using
+/// the Microsoft <c>System.Text.Json</c> library
 /// </summary>
-internal sealed class LowLevelRequestResponseSerializer : Serializer
+internal sealed class LowLevelRequestResponseSerializer :
+	SystemTextJsonSerializer
 {
 	/// <summary>
 	/// Provides a static reusable reference to an instance of <see cref="LowLevelRequestResponseSerializer"/> to promote reuse.
 	/// </summary>
 	internal static readonly LowLevelRequestResponseSerializer Instance = new();
 
-	private readonly Lazy<JsonSerializerOptions> _indented;
```

Comment:
Lazy initialization is provided by the `SystemTextJsonSerializer` base class (in a slightly different way).

Reply:
I wonder why we even make this Lazy to begin with.

Reply:
Can we get rid of this complexity (and `Initialised`) by just doing this eagerly in the constructor?

Reply:
I vaguely remember an edge case that can happen when calling an abstract/virtual method in the constructor. Thought that was the reason for the lazy initialization.

I'll double check the implications and if there are none, I'll happily get rid of that extra complexity.

Reply:
Ah, I remember: https://learn.microsoft.com/de-de/previous-versions/visualstudio/visual-studio-2015/code-quality/ca2214-do-not-call-overridable-methods-in-constructors?view=vs-2015&redirectedfrom=MSDN

This will definitely be a problem.